project-base/.ci/docker/templates/nginx.conf

```diff
@@ -0,0 +1,15 @@
+server {
+
+        listen 80;
+        server_name ~^(\d+\.)?{{JOB_NAME}}\.{{DEVELOPMENT_SERVER_DOMAIN}}$;
+
+        location / {
+                proxy_pass http://{{PORT_WEB}};
```

Comment:
are you sure this is correct?

Reply:
yes, `/usr/local/bin/docker-compose port webserver 8080` returns ip with port. defaultly it is `0.0.0.0:8080`.